xbmc/guilib/GUIVideoControl.cpp

```diff
@@ -38,6 +38,9 @@ void CGUIVideoControl::Process(unsigned int currentTime, CDirtyRegionList &dirty
 
 void CGUIVideoControl::Render()
 {
+  if (CServiceBroker::GetWinSystem()->GetGfxContext().GetRenderOrder() ==
+      RENDER_ORDER_FRONT_TO_BACK)
+    return;
```

Comment:
See a bit lower in GUIVideoControl::Render, it seems to have alpha like other controls. I don't know skinning and am going to guess but it looks like it could be overlaid on something else (maybe with tweened alpha for example).
That would mean it should have a proper depth and not be systematically at the back. If alpha is 255 then it should probably be part of the front to back pass.

Reply:
It has to be in the back to be consistent across all platforms. When doing hardware rendering on ARM, the HW compositor draws the video layer below the GUI layer. It can't be somewhere in between GUI elements*.

Reply:
I'm not sure about the alpha part, but I think that's a relic from the past. We can't have alpha for video layers on some ARM devices.

Reply:
And it should be the first element in the back to front pass. We know that it is the furthest element to render, so we don't need to update the depth buffer.

Reply:
*to display the video "between" layers on ARM systems, we could draw an placeholder element where the video "shines through" from below.